specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -214,7 +214,7 @@
             "required": true,
             "x-ms-client-flatten": true,
             "schema": {
-              "$ref": "#/definitions/NameAndUserDataContract"
+              "$ref": "#/definitions/NameAndUserDataAndModelTypeContract"
```

Comment:
We need to create two definitions:
1. The original `NameAndUserDataContract`
2. `MetaDataContract` inheriting from `NameAndUserDataContract`, adding `recognitionModel` as property.

Person do not need `recognitionModel` in our design.

Reply:
Done.